### ISSUE TITLE
[Bug] Behavior 'AllowInjectionScopeLocation' doesn't make 'IInjectionScope' locatable thru Locate<IInjectionScope>

### ISSUE THREAD

**Issue by mcdis** [ipjohnson/Grace#172]
```
Container = new DependencyInjectionContainer(_ =>
{
  _.Behaviors.AllowInjectionScopeLocation = true;
});
Container.Locate<IInjectionScope>(); <--- Fail
```

**Comment by ipjohnson** [comment]
I believe I know why it’s not returning itself. I’ll look to make this change in the coming days and release a beta with the other issue you raised

**Comment by ipjohnson** [comment]
I've released 6.4.0-RC685 with this fix
